CHANGELOG.md

```diff
@@ -55,6 +55,10 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 
 ##### Bug Fixes
 
+* fix pod lib lint errors for pod with test_spec
```

Comment:
please be more descriptive on this change. "Generate `Info.plist` for test targets"` for example.

Also update your commit message to reflect this similarly.

Reply:
yes, great idea, added more details.